app/models/concerns/hyrax/collection_behavior.rb

```diff
@@ -55,19 +55,22 @@ def add_members(new_member_ids)
       members << ActiveFedora::Base.find(new_member_ids)
     end
 
+
     # Add member objects by adding this collection to the objects' member_of_collection association.
     def add_member_objects(new_member_ids)
+      error_messages = ["THis is just a test"]
       Array(new_member_ids).each do |member_id|
         member = ActiveFedora::Base.find(member_id)
-        # @note Ideally, this would be surfaced as a warning in a flash
-        #       message. Because the member is found and saved in this model
-        #       method, I am not sure it's worth the effort to rejigger things
-        #       such that this information bubbles up to the controller and
-        #       view.
-        next if Hyrax::MultipleMembershipChecker.new(item: member).check(collection_ids: id, include_current_members: true)
+        #will return and error message as a string, otherwise it will return a nil
+        message = Hyrax::MultipleMembershipChecker.new(item: member).check(collection_ids: id, include_current_members: true)
+        if message
+           error_messages << message
+           next
+        end
         member.member_of_collections << self
         member.save!
       end
+      return error_messages
```

Comment:
I think the return behavior we want is to return `error_messages` if it's got anything in it (check `error_messages.any?` for instance), otherwise return the result of `member.save!`.

Reply:
I think we always want to return error_messages. In the collection controller it will check  error_messages.any? in order to add to the flash.  Right?  That's the way I read it.